src/Randomizer.Multiplayer.Server/README.md

```diff
@@ -0,0 +1,95 @@
+# SMZ3 Cas’ Multiplayer Server
```

Comment:
Is this document intended for other developers or potential server admins? It seems the information is a little mixed; if it's intended for consumers then this is probably better placed in the general README (or a separate document, e.g. in a new `docs\` folder in the repo root that's linked from the root README).

Maybe it's worth splitting up this info?

Reply:
I'll move this over to a docs folder and link it out from the main repo since it's more for server admins, yeah.

Reply:
Admittedly part of the reason I included the build info is I'm not exactly sure of the best way of providing server builds to folks. Though, I suppose we could include the server download with each release regardless of whether it's updated or not? I'm not sure. What would be nice is if we could have two lists of releases, one for the server and one for the actual randomizer/tracker, but I don't think github really is setup for that, is it?